docs/docs/errors.md

```diff
@@ -136,6 +136,8 @@ The `callbackUrl` provided was either invalid or not defined. See [specifying a
 
 #### JWT_SESSION_ERROR
 
+JWEDecryptionFailed: NextAuth needs NEXTAUTH_SECRET environment variable to encrypt JWTs and to hash email verification tokens. 
```

Comment:
```suggestion
JWEDecryptionFailed: NextAuth.js needs `NEXTAUTH_SECRET` environment variable to encrypt JWTs and to hash email verification tokens. This can also occur if you have changed the `NEXTAUTH_SECRET`, but you still had an active session with the old secret. Logging in again solves the issue.
```